alias

```diff
@@ -134,3 +134,13 @@ revoke-my-ip-all =
   !f() {
     aws revoke-my-ip ${1} all all
   }; f
+
+get-vpc-id-by-name =
+  !f() {
+      aws ec2 describe-vpcs --filter "Name=tag:Name,Values=${1}" --query 'Vpcs[*].{id:VpcId}' --output text
```

Comment:
I think you should be able to simplify your ``--query`` by just making it: ``--query 'Vpcs[].[VpcId]`` and should be the same result

Reply:
@kyleknap the alias as submitted is correct.

```console
$ aws ec2 describe-vpcs --filter "Name=tag:Name,Values=vpc_name" --query 'Vpcs[].[id:VpcId]'

Bad value for --query Vpcs[].[id:VpcId]: Expecting: comma, got: colon: Parse error at column 10, token ":" (COLON), for expression:
"Vpcs[].[id:VpcId]"
           ^
$ aws ec2 describe-vpcs --filter "Name=tag:Name,Values=vpc_name" --query 'Vpcs[].{id:VpcId}'
[
    {
        "id": "vpc-01234567890123456"
    }
]